src/config/services/managing.md

```diff
@@ -70,6 +70,18 @@ currently running:
 # rm /etc/runit/runsvdir/default/<service>
 ```
 
+### Testing services
+
+To check a service works as expected before fully enabling it:
```

Comment:
Perhaps explain that this is necessary for the service to run in the environment it is expected to run?

Reply:
Hmm, yes. Any suggested wording?

Reply:
"To check if a service is working correctly when started by the service supervisor, it is possible to run it once, before fully enabling it:"

Reply:
Would you be okay with: "To check if a service is working correctly when started by the service supervisor, run it once before fully enabling it:"?

Reply:
yup!

Reply:
Done. :-)